### DISCUSSION TITLE
Module not found:  contains a reference to the file "react/jsx-runtime"

### DISCUSSION THREAD

**Discussion by myr45500** [family/connectkit#159] [Q&A] [answered]
i installed the package in an existing minting dapp and when i run the local server i get this error.

Module build failed: Module not found:
"./node_modules/connectkit/build/index.es.js" contains a reference to the file "react/jsx-runtime"

React 7.0.2
Typescript

any suggestions. thanks

**Comment by lochie** [comment, marked as answer]
You'll most likely need to update to React `18` and typescript `4.9.4`
